packages/runtime/test/src/logicsig.ts

```diff
@@ -54,20 +61,31 @@ describe("Logic Signature", () => {
 
 describe("Multi-Signature Test", () => {
   useFixture("multi-signature");
-  const alice = new AccountStore(10);
-  const john = new AccountStore(100);
-  const bob = new AccountStore(1000);
-  const bobPk = decodeAddress(bob.address).publicKey;
-
-  const runtime = new Runtime([alice, john, bob]);
-  // Generate multi signature account hash
-  const addrs = [alice.address, john.address, bob.address];
-  const mparams = {
-    version: 1,
-    threshold: 2,
-    addrs: addrs
-  };
-  const multsigaddr = multisigAddress(mparams);
+  let alice: AccountStore;
+  let john: AccountStore;
+  let bob: AccountStore;
+  let runtime: Runtime;
+  let bobPk: Uint8Array;
+  let mparams: MultisigMetadata;
+  let multsigaddr: string;
+
+  before(() => {
+    alice = new AccountStore(10);
+    john = new AccountStore(100);
+    bob = new AccountStore(1000);
+    bobPk = decodeAddress(bob.address).publicKey;
+
+    runtime = new Runtime([alice, john, bob]);
+
+    // Generate multi signature account hash
+    const addrs = [alice.address, john.address, bob.address];
+    mparams = {
+      version: 1,
+      threshold: 2,
+      addrs: addrs
+    };
+    multsigaddr = multisigAddress(mparams);
+  });
```

Comment:
why doing this change?

Reply:
OK, thanks. Please add a comment to the code that we have initialization dependency with loading env.

Reply:
sure